WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +709,32 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
+            if (isDraft) {
+                if (PostUtils.isPublishable(post)) {
```

Comment:
> This check isn't necessary,

I'm not sure about this one  - I addressed it almost blindly in 587e005  but now that I'm thinking of it, the checks are there because we are offering the `PUBLISH` button in that snackbar if the post is publishable. With this last commit in place, we'd be now offering the possibility to tap `PUBLISH` but they can't actually publish anything ([they'd immediately get a Toast indicating the post can't be published](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/async-media-snackbars/WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java#L564-L568)).

Because of this, I'm keen to reverting this commit if you agree. Let me know.